src/compat/dbtype.h

```diff
@@ -772,6 +772,7 @@ union db_char
   {
     unsigned char style;
     unsigned char codeset;
+    bool compare_LT;
```

Comment:
compare_LT is not very clear as it is. I think it is more clear if we'd name it is_max_string. is_max_string we know must be bigger than anything.